spec/jarvis/location_times_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "jarvis/location_times"
+
+describe Jarvis::LocationTimes do
+  let(:utc) { Time.new(2016, 1, 8, 16, 30, 00).utc }
```

Comment:
I think the tests fail for me because Time.new is created in local time before conversion to UTC.

```
>> Time.new(2016, 1, 8, 16, 30, 00).utc
=> 2016-01-09 00:30:00 UTC
```